include/lsst/jointcal/CcdImage.h

```diff
@@ -20,8 +20,8 @@
 namespace lsst {
 namespace jointcal {
 
-typedef int ShootIdType;
-
+typedef int VisitIdType;
+typedef int CcdIdType;
```

Comment:
Scratch that: it looks like that may confuse SWIG. I'll leave them as typedefs until we pybind11 jointcal.